src/pyscipopt/expr.pxi

```diff
@@ -111,6 +186,11 @@ cdef class Expr:
         elif _is_number(other):
             c = float(other)
             self.terms[CONST] = self.terms.get(CONST, 0.0) + c
+        elif isinstance(other, GenExpr):
+            # is no longer in place, might affect performance?
+            # can't do `self = buildGenExprObj(self) + other` since I get
+            # TypeError: Cannot convert pyscipopt.scip.SumExpr to pyscipopt.scip.Expr
+            return buildGenExprObj(self) + other
```

Comment:
what if we do this instead:

    return other + buildGenExprObj(self)

Would this be inplace (but for other's place)?

Reply:
I am guessing that inplace means self's place. But yeah, this inplace stuff is a bit complicated and would also postpone it.